### DISCUSSION TITLE
What animals recently helped police officers find a criminal in NC?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#850] [Galaxy Trivia] [answered]
*answer with emoji*

**Comment by peterbaccaro** [comment, marked as answer]
🐮 🐮 🐮 A herd of cows helped a North Carolina police department track a suspect who fled from officers during a traffic stop on Tuesday 9th May 2023, allowing police to locate and arrest the suspect.
